apps/web/.env.sample

```
@@ -27,11 +27,12 @@ SMTP_USER=
 SMTP_PWD=
 
 # Object Storage (S3) (optional)
-# Defaults below are configured for the local MinIO instance (see docker-compose.dev.yml)
+# Defaults below are configured for the local Garage instance (see docker-compose.dev.yml)
 S3_BUCKET_NAME=rallly
-S3_ENDPOINT=http://0.0.0.0:9000
-S3_ACCESS_KEY_ID=minio
-S3_SECRET_ACCESS_KEY=minio123
+S3_ENDPOINT=http://localhost:3900
+S3_ACCESS_KEY_ID=rallly-access-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_ACCESS_KEY_ID key should go before the S3_BUCKET_NAME key

(UnorderedKey)